scalability_and_performance/recommended-host-practices.adoc

```diff
@@ -29,6 +29,11 @@ include::modules/increasing-aws-flavor-size.adoc[leveloffset=+2]
 
 include::modules/recommended-etcd-practices.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Needs an ID: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Suggested edit:
```suggestion
[id="additional-resources_recommended-etcd-practices"]
.Additional resources
```

Reply:
> Needs an ID: [https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?rgh-link-date=2022-07-07T17%3A09%3A55Z#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)
> 
> Suggested edit:

Hi @jldohmann Thanks for your comments! For this one, the 'role=' statements is needed, yes? That is what I see in the manual.

Reply:
@tmalove yep! Both the role and ID are needed